apps/studio/src/pages/dashboard/edit/index.tsx

```diff
@@ -195,7 +205,13 @@ const EditPage: NextPageWithLayout = () => {
       </GridItem>
       {/* TODO: Implement preview */}
       <GridItem colSpan={2} overflow="scroll">
-        <Preview schema={editedSchema} />
+        <Preview
+          schema={{
+            content: pageState,
+            version: page.version,
```

Comment:
note: out of scope of this PR but just FYI, the version here should take the version of the schema being used, not the existing version of the page. This is because we should try to "upgrade" the page as much as possible and prevent the editor from saving if the contents of the page violates a future version of the schema.

Reply:
agree wth you, this is abit annoying to do at present cos of how the query itself is structured. lets do it in a follow-up PR since for now, there's only 1 versoin ._.

Reply:
added a todo to track this